### PR TITLE
Modal rating typography

### PR DESCRIPTION
**Pull request by deekshithmd**
Added modal, rating and typography components

### CODE REVIEW

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.rating .fas{
+    color:var(--secondary-color);
```
Comment (anshulraheja) [inline components/rating/rating.css:2]: add cursor pointer property too

**Review by anshulraheja — APPROVED** [review]
The code looks good to merge!

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: components/rating/rating.css (hunk 1 above)
Comment (deekshithmd) [reply components/rating/rating.css:2]: ok, I will check that. Thank you for the review

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.modal-container{
```
Comment (mayuresh-tech) [inline components/modal/modal.css:1]: Suggestion (do later):
Make background non-scrollable when modal is shown

Else everything looks good!
Good to merge!

**Pull request merged**